src/utils/listOfTests.js

```diff
@@ -0,0 +1,350 @@
+export const typeOfMedicalExamination = [
+    {
```

Comment:
Maybe change props name to: medicExamName, medicExamKey?

Reply:
And maybe stick to one name of array? I mean that youve added other file name and other name of array. Same above.